examples/Active Inference Mountain car.ipynb

```diff
@@ -1,37 +1,25 @@
 {
```

Comment:
<p>As a reader, I would not really understand this equation. It is called "the gravitational force" while, in fact, I'm pretty much sure it has little to do with Newton's laws. It seems to be that this is equation is a combination of the landscape and associated forces. As a reader I also would not understand what is the argument "\phi" of the function. Please add more explanation of this equation.</p><p> </p><p>In the <code>create_physics</code> function we also have two extra forces, which are not explained much. We could add some brief explanation to the friction force and to the engine force functions as well.  Should not be very detailed, just a brief explanation would be enough.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/biaslab/RxInfer.jl/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
@Sepideh-Adamiat @HoangMHNguyen